src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -2369,14 +2354,8 @@ void SlurTieLayout::computeBezier(SlurSegment* slurSeg, PointF shoulderOffset)
     slurSeg->ups(Grip::SHOULDER).p = toSystemCoordinates.map(p6);
 
     // Set slur thickness
-    double w = slurSeg->style().styleMM(Sid::SlurMidWidth) - slurSeg->style().styleMM(Sid::SlurEndWidth);
-    if (slurSeg->staff()) {
-        w *= slurSeg->staff()->staffMag(slurSeg->slur()->tick());
-    }
-    if ((c2 - c1) <= _spatium) {
-        w *= .5;
-    }
-    PointF thick(0.0, w);
+    computeMidThickness(slurSeg, p2.x() / slurSeg->spatium());
```

Comment:
This would conflict with https://github.com/musescore/MuseScore/pull/20693. Not a big problem, but one PR or the other will need to be adjusted when the other is merged.

(What's dangerous, is that it won't be a conflict that Git can detect, since the conflicting changes are in different places)

Reply:
Well, _the other_ ;)